server/app/views/admin/programs/ProgramPredicatesEditView.java

```diff
@@ -298,6 +312,45 @@ private static LabelTag renderPredicateQuestionCheckBoxRow(
                         .withClasses("mt-1", "text-sm")));
   }
 
+  private DivTag createEligibilityMessageForm(
+      Http.Request request, BlockDefinition block, ProgramDefinition program) {
+    String existingEligibilityMessage =
+        block.localizedEligibilityMessage().map(LocalizedStrings::getDefault).orElse("");
+    BlockEligibilityMessageForm existingEligibilityMessageForm =
+        new BlockEligibilityMessageForm(existingEligibilityMessage);
+    Form<BlockEligibilityMessageForm> form =
+        formFactory.form(BlockEligibilityMessageForm.class).fill(existingEligibilityMessageForm);
+    DivTag buttonDiv =
+        div()
```

Comment:
**nitpick:** Do we need to wrap the button in an empty `<div>`?

Reply:
Gotcha, making it a ButtonTag instead of a Divtag.